cpp/tests/utilities/timestamp_utilities.cuh

```diff
@@ -60,9 +60,10 @@ generate_timestamps(int32_t count, time_point_ms start, time_point_ms stop) {
                  .time_since_epoch()
                  .count();
 
+  // When C++17, auto [min, max] = std::minmax(lhs, rhs)
   auto min = std::min(lhs, rhs);
   auto max = std::max(lhs, rhs);
-  auto range = static_cast<Rep>(std::abs(max - min));
+  auto range = static_cast<Rep>(max - min);
```

Comment:
Since start can be larger than stop, you need the abs to guarantee a positive range.

Reply:
Isn't that what the `std::min` and `std::max` calls with `lhs` and `rhs` take care of?

Reply:
Ah yes. Right. But in general this is a nitpick PR that doesn't fix a correctness or performance problem. The review discussion cost is higher than any benefit. Suggest incorporating changes like this along with other changes.

Reply:
Good point, will do in the future